lib/rails_multisite/formatter.rb

```diff
@@ -0,0 +1,13 @@
+module RailsMultisite
+  class Formatter < ::ActiveSupport::Logger::SimpleFormatter
+    def call(severity, timestamp, progname, msg)
+      string = super
+
+      if RailsMultisite::ConnectionManagement.all_dbs != [RailsMultisite::ConnectionManagement::DEFAULT]
```

Comment:
why introduce the formatter at all if its not running multisite? no need to have the check and make arrays each time, only introduce the formatter if it truly is multisite.